src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/RSBridgePeripheral.java

```diff
@@ -0,0 +1,905 @@
+            return MethodResult.of(0, "Target Tank does not exist");
+
+        return MethodResult.of(ChemicalUtil.moveChemical(chemicalHandler, targetTank, filter.getLeft()));
+    }
+
+    /**
+     * imports an item to the system from a valid inventory
+     *
+     * @param arguments       the arguments given by the computer
+     * @param targetInventory the give inventory
+     * @return the imported amount or null with a string if something went wrong
+     */
+    protected MethodResult importToRS(@NotNull IArguments arguments, @Nullable IItemHandler targetInventory) throws LuaException {
+        RsItemHandler itemHandler = new RsItemHandler(getNetwork());
+        Pair<ItemFilter, String> filter = ItemFilter.parse(arguments.getTable(0));
+
+        if (filter.rightPresent())
+            return MethodResult.of(0, filter.getRight());
+
+        if (targetInventory == null)
+            return MethodResult.of(0, "Target Inventory does not exist");
+
+        return MethodResult.of(InventoryUtil.moveItem(targetInventory, itemHandler, filter.getLeft()));
+    }
+
+    /**
+     * imports a fluid to the system from a valid tank
+     *
+     * @param arguments  the arguments given by the computer
+     * @param targetTank the give tank
+     * @return the imported amount or null with a string if something went wrong
+     */
+    protected MethodResult importToRS(@NotNull IArguments arguments, @Nullable IFluidHandler targetTank) throws LuaException {
+        RsFluidHandler fluidHandler = new RsFluidHandler(getNetwork());
+        Pair<FluidFilter, String> filter = FluidFilter.parse(arguments.getTable(0));
+
+        if (filter.rightPresent())
+            return MethodResult.of(0, filter.getRight());
+
+        if (targetTank == null)
+            return MethodResult.of(0, "Target Tank does not exist");
+
+        return MethodResult.of(FluidUtil.moveFluid(targetTank, fluidHandler, filter.getLeft()));
+    }
+
+    /**
+     * imports a fluid to the system from a valid tank
+     *
+     * @param arguments  the arguments given by the computer
+     * @param targetTank the give tank
+     * @return the imported amount or null with a string if something went wrong
+     */
+    protected MethodResult importToRS(@NotNull IArguments arguments, @Nullable IChemicalHandler targetTank) throws LuaException {
+        RsChemicalHandler chemicalHandler = new RsChemicalHandler(getNetwork());
+        Pair<ChemicalFilter, String> filter = ChemicalFilter.parse(arguments.getTable(0));
+
+        if (filter.rightPresent())
+            return MethodResult.of(0, filter.getRight());
+
+        if (targetTank == null)
+            return MethodResult.of(0, "Target Tank does not exist");
+
+        return MethodResult.of(ChemicalUtil.moveChemical(targetTank, chemicalHandler, filter.getLeft()));
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult isConnected() {
+        return MethodResult.of(isAvailable());
```

Comment:
if only one variable is always returned, I think it is not necessary to return a MethodResult